src/Compilers/CSharp/Portable/PublicAPI.Shipped.txt

```diff
@@ -161,9 +161,12 @@ Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.Body.get -> Micro
 Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.Keyword.get -> Microsoft.CodeAnalysis.SyntaxToken
 Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.Modifiers.get -> Microsoft.CodeAnalysis.SyntaxTokenList
 Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.SemicolonToken.get -> Microsoft.CodeAnalysis.SyntaxToken
+Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.ExpressionBody.get -> Microsoft.CodeAnalysis.CSharp.Syntax.ArrowExpressionClauseSyntax
```

Comment:
@gafter I had to add `overwrite` in `PublicAPI.Shipped.txt``` for i.a.
```
Microsoft.CodeAnalysis.CSharp.Syntax.ConversionOperatorDeclarationSyntax.ExpressionBody.get -> Microsoft.CodeAnalysis.CSharp.Syntax.ArrowExpressionClauseSyntax
```
Is this API breaking? Deleted the corresponding lines in Shipped.txt and added them to Unshipped.txt